rakelib/artifacts.rake

```diff
@@ -103,6 +103,23 @@ namespace "artifact" do
     @exclude_paths << 'vendor/jruby/lib/ruby/gems/shared/gems/rexml-3.2.5/**/*'
     @exclude_paths << 'vendor/jruby/lib/ruby/gems/shared/specifications/rexml-3.2.5.gemspec'
 
+    # remove this after JRuby includes cgi-0.3.7+
+    @exclude_paths << 'vendor/jruby/lib/ruby/gems/shared/specifications/default/cgi-0.3.6-java.gemspec'
+    @exclude_paths << 'vendor/jruby/lib/ruby/stdlib/cgi{,.rb}'
+    @exclude_paths << 'vendor/jruby/lib/ruby/stdlib/cgi/**/*'
```

Comment:
Review note:

1. build a tarball:
   ~~~
   ./gradlew assembleTarDistribution
   ~~~
2. Look for instances of the word `cgi`, and ensure that the stdlib entry present in `vendor/jruby/lib/ruby/stdlib` of source is not present in the tarbal:
   ~~~
   tar -tzf build/logstash-9.1.0-SNAPSHOT.tar.gz | egrep '\cgi\b'
   ~~~